spec/models/info_request_spec.rb

```diff
@@ -71,6 +71,13 @@
       expect(info_request.url_title).to eq("test_title_3")
     end
 
+    it "strips line breaks from the title" do
+      info_request = FactoryGirl.create(:info_request,
+                                       :title => "Title\rwith\nline\r\nbreaks")
+      info_request.save
```

Comment:
bah, I originally had `FactoryGirl.build` and changed my mind - forgot to take out the accompanying `save`

thanks, will fix!